src/BehaviorsSDKManaged/BehaviorsSDKManaged.sln

```diff
@@ -253,6 +251,46 @@ Global
 		{7119F232-7BEF-42F9-9BA1-52654ADC98B6}.Release|x64.Build.0 = Release|x64
 		{7119F232-7BEF-42F9-9BA1-52654ADC98B6}.Release|x86.ActiveCfg = Release|x86
 		{7119F232-7BEF-42F9-9BA1-52654ADC98B6}.Release|x86.Build.0 = Release|x86
+		{E6A328C8-2D9A-459A-86EE-F1C50CC6DADD}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{E6A328C8-2D9A-459A-86EE-F1C50CC6DADD}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{E6A328C8-2D9A-459A-86EE-F1C50CC6DADD}.Debug|ARM.ActiveCfg = Debug|Any CPU
```

Comment:
These ARM configurations look wrong. They should probably match what the x64 arch is doing, no?

Reply:
Yeah I agree, but for some reason the WinUI projects were also using this weird mix, so I just did the same here for consistency. Should we fix this in both places? I do have to say it doesn't seem right at all 🤔

Reply:
I would probably just remove it, as we don't support this build config at all, or at least we shouldn't.